python/nvforest/tests/test_benchmark.py

```
@@ -0,0 +1,716 @@
+        try:
+            model = train_model(
+                FRAMEWORKS["lightgbm"],
+                model_type="regressor",
+                num_trees=5,
+                max_depth=3,
+                X_train=X,
+                y_train=y,
+                device="gpu",
+            )
+            assert hasattr(model, "predict")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)